docs/application/dotnet/guides/app-management/application-launcher.md

```diff
@@ -0,0 +1,304 @@
+# Application launcher
+
+This guide explains how to create a basic application launcher. The application launcher is a main application that normally starts after system boot. This app is commonly replaced in the platform adjustments process. It is easy to use the .NET APIs in the application launcher implementation.
+
+## Application launcher functionalities
+
+Every application launcher must be able to:
+ - List installed and runnable applications.
+ - Run selected application by the user.
+
+You can also implement advanced functionalities such as:
+ - grouping installed applications into folders,
+ <!-- TBD: when widgets viewer will be pushed unlock this line - [widgets viewer](./widgets_viewer.md), -->
+ - removing applications,
+ - rearrangement of icons,
+ - viewing notifications,
+
+## Prerequisites
+
+To list the installed applications and launch them, the launcher app must have defined privileges in the project manifest file as:
+
+**Snippet: Application launcher priviledges**
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+  <ui-application appid="NUIApplicationLauncher">
+    <!-- Auto generated manifest properties -->
+  </ui-application>
+  <privileges>
+    <privilege>http://tizen.org/privilege/packagemanager.info</privilege>
+    <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+  </privileges>
+```
+
+## Application launcher sample code
+
+Following code snippet illustrates how to implement the simple application launcher using:
```

Comment:
Is this an example specific to NUI, package Manager, TizenFX Appcontrol or does is generic step to app launcher that every developer must implement?

Reply:
This is related more to TizenFX API's but NUI is used as a main UI framework